_drafts/uw-capstone.md

```diff
@@ -0,0 +1,111 @@
+
+And yes, we did discuss just how misleading it can be to rely on the term “feature importance” when communicating with people about fitted models. But more than anything, we discussed how the students’ work could best serve not just the immediate questions at hand, but also later questions not yet explored or even asked. We began to consider a broader notion of our collaboration, one that would include future unknown partners and reap the benefits of groundwork begun now. The deliverables for this project would include the power to make future projects more fruitful themselves.
+
+## Results and the Future
+
+With such a challenging dataset and problem space, no one involved in the project was expecting the results to be tidy or simple. In their report on the project, the students focused on the future value of their work, writing, "The biggest challenges with the SIMS data are the knowledge barrier for the clinical data and the structure of the raw survey-related data. The significant amount of work that has gone into creating the data pipeline, data mapping, and reproducible analyses will be useful tools in generating more key insights from the SIMS data." 
+
+Reading the students’ description, I noted their emphasis on knowledge of clinical data. There is a canonical structure to images used when defining “data science.” If you’ve done reading on data science, you might have seen it. I took this one from [Towards Data Science](https://towardsdatascience.com/introduction-to-statistics-e9d72d818745) ![Data Science Venn Diagram]({{ site.baseurl }}/assets/images/data_science_venn_diagram.png) There are plenty of variations, but that structure generally puts “data science” at the central intersection of a Venn diagram of Math/Statistics, Computing/Code, and Domain Knowledge. The implication of that last circle is important: If a data scientist doesn’t know enough about the specific domain where she is working, how can she expect to understand and evaluate the implications of her choices, assumptions, and results? There are plenty of wrong choices in data science, but “wrong” isn’t really at the crux. Instead, most choices, assumptions, and results are wrong/right/wise/foolish/meaningful/etc. within the context of a specific domain. If you’re forced to make choices without the benefit of appropriate domain knowledge, it’s fair to say that you are, at least to an extent, flying blind.
```

Comment:
Have you rendered this locally? I think the image will need to be on a separate line.

Reply:
I haven't but I was going to add the line breaks in just a moment!